video/out/opengl/context_drm_egl.c

```diff
@@ -100,20 +100,51 @@ static const char *gbm_format_to_string(uint32_t format)
     }
 }
 
+// Allow falling back to an ARGB EGLConfig when we have an XRGB framebuffer.
+// Also allow falling back to an XRGB EGLConfig for ARGB framebuffers, since
+// this seems neccessary to work with broken Mali drivers that don't report
+// their EGLConfigs as supporting alpha properly.
+static uint32_t fallback_format_for(uint32_t format) {
+    switch (format) {
+        case GBM_FORMAT_XRGB8888:
```

Comment:
The whole function suddenly changes coding style wrt. whitespace in the function decl. and the switch statement.

Otherwise LGTM.

Reply:
derp! fixed

Reply:
Sorry, still not right... we don't indent the case labels in switch statements.

Reply:
Is this good? (I also threw in a correction of the indentation of the gbm_format_to_string function)

Reply:
Yeah.